QgisModelBaker/utils/gui_utils.py

```diff
@@ -204,18 +204,15 @@ def get_text_color_object(level: LogLevel = LogLevel.INFO) -> QColor:
     return QColor(get_text_color(level))
 
 
-def get_parsed_log_text_color(text, txt_edit) -> None:
+def get_parsed_log_text_level(text) -> LogLevel:
     textlines = text.splitlines()
```

Comment:
It seems a bit strange to deal with several lines here but only return the first line case.

If we don't have a multifaceted case where we get a `textlines` variable with several levels (i.e., warning + error, info + warning), please add a comment to explain that we'll get the level from the first line content.